docs/core/howto/tap.rst

```diff
@@ -310,6 +310,107 @@ For a full list of cred plugins supported, see :api:`twisted.plugins <twisted.pl
 
 
 
+Deploy your application using python packages
+---------------------------------------------
+
+
+
+To deploy your application one possibility is to wrap it up in a python package.
+For this you need to write a special file ``setup.py``, which contains metadata
+of the package. You would have to extend the layout of your files like this:
```

Comment:
The directory layout for plugins is described in the plugins document.  If those docs aren't clear, perhaps they should be improved.  Then this document could refer to that one.